templates/haproxy.cfg.j2

```diff
@@ -0,0 +1,35 @@
+global
+        maxconn 100
+
+defaults
+        log     global
+        mode    tcp
+        retries 2
+        timeout client 30m
+        timeout connect 4s
+        timeout server 30m
+        timeout check 5s
+
+listen stats
+        mode http
+        bind *:{{ patroni_haproxy_stats_listen_port }}
+        stats enable
+        stats uri /
+
+listen master
+        bind *:{{ patroni_haproxy_leader_listen_port }}
+        option httpchk OPTIONS /master
+        http-check expect status 200
+        default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
+{% for server in patroni_haproxy_servers %}
+        server {{ server.name }} {{ server.ip }}:{{ server.port }} maxconn 100 check port 8008
```

Comment:
It would be great to avoid hardcoded values for the REST API port here. Either by getting the port value from the `patroni_restapi_listen` or introducing a new variable (e.g. `patroni_restapi_port`).

Reply:
Went with the second option and introduced `patroni_restapi_port`. Since it's used only to generate other variables, it shouldn't introduce any breaking change.